defi.tokenlist.json

```diff
@@ -595,6 +602,7 @@
           "symbol": "RAI",
           "decimals": 18,
           "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/0x97Cd1CFE2ed5712660bb6c14053C0EcB031Bff7d/logo.png"
+>>>>>>> main
```

Comment:
This shouldn't be here. Looks like it's left behind from some previous merge conflict resolution?